ddtrace/internal/datadog/profiling/stack_v2/__init__.pyi

```diff
@@ -2,19 +2,39 @@ import asyncio
 from types import FrameType
 from typing import Optional, Sequence, Union
 
-def register_thread(id: int, native_id: int, name: str) -> None: ...  # noqa: A002
+from ddtrace._trace import context
+from ddtrace._trace import span as ddspan
+
+# Core stack v2 functions
+def start(min_interval: float = ...) -> bool: ...
+def stop() -> None: ...
+
+# Sampling configuration
+def set_adaptive_sampling(do_adaptive_sampling: bool = False) -> None: ...
```

Comment:
Should this take a target for the adaptive sampling?

Reply:
Filed https://datadoghq.atlassian.net/browse/PROF-13057